README.md

```diff
@@ -43,7 +43,9 @@ and then:
 pod install
 ```
 
-to install your dependencies
+to install your dependencies.
+
+Finally, inside Xcode, go to the Capabilities section for your target and switch Push Notifications on.
```

Comment:
No, it's Xcode 8 requirement, and we have to use Xcode 8 as we are using Swift 3 now